tests/tests_async.py

```diff
@@ -94,3 +95,13 @@ def test_async_call_with_defaults(self):
             lambda_function_name="MyLambda",
         )
         lambda_async_mock.return_value.send.assert_called_with(get_func_task_path(async_me), ("qux",), {})
+
+    def test_async_call_arg_not_json_serializable(self):
+        """Exception is raised when calling an async function locally (not on aws)"""
+        async_me = import_and_get_task("tests.test_app.async_me")
+        unserializable_object = datetime.datetime.now()
+        with self.assertRaises(TypeError):
+            async_me(unserializable_object)
+
+        with self.assertRaises(UnserializableJsonError):
+            async_me(unserializable_object)
```

Comment:
```suggestion
        with self.assertRaises(UnserializableJsonError):
            async_me(unserializable_object)
```

Am I missing a reason that the latter assertion is not sufficient by itself?

Having both would seem to just be ensuring `UnserializableJsonError` is a subclass of `TypeError`, which doesn't seem too useful.

IMO, our unit tests should assume that Python internals (ie. class inheritance, the fact that `except ExceptionClass:` catches `ExceptionClass` and all of its subclasses, etc.) work they way they are documented (and are thoroughly tested for us already, in the unit tests of `CPython` 🙂).

Reply:
Thanks for the review.
Yes, the `TypeError` check is unnecessary.  I've committed the suggestion!